patches/common/chromium/woa_compiler_workaround.patch

```diff
@@ -0,0 +1,37 @@
+From b3414d055399d0a21f6166a536467ea752b2aa8a Mon Sep 17 00:00:00 2001
```

Comment:
Just FYI @jkleinsc @felixrieseberg @richard-townsend-arm this patch file is invalid.  Although it works it wasn't generated by the `export-patches` script which means that the next time someone runs import/export on their patch directory this file will change.  E.g. The `From` line here should be `000000000...` and the git version `2.19.1.windows.1` shouldn't be present.

`./electron/script/git-export-patches -o ./electron/patches/common/chromium` is the command to export a patch applied on top of chromium 👍

Reply:
Sounds like the kind of thing a presubmit hook should check for :D